src/mudlet.cpp

```diff
@@ -2278,6 +2291,13 @@ void mudlet::forceClose()
         console->close();
     }
 
+    for (auto& hostName: mudlet::self()->getHostManager().getHostList()) {
```

Comment:
This is identical with a new block of code above - is it worth `refactor`ing as "extract as a function" (or sommat like that as the Qr Creator refactor context menu item has it) to a private method.